content/en/docs/Getting started/_index.md

```diff
@@ -5,23 +5,122 @@ weight: 100
 description: Quick Baker setup instructions
 ---
 
-Baker is a Go library and it must be included into your Go program.
+Do you think Baker suits your needs? Good! Now let's see together how to start using it.
```

Comment:
I like a more conversational/informal tone, however with this sentence it seems we are asking the user confimation if they really want to use baker. Proposition:
```suggestion
Looking to use Baker and start building pipelines now? Great, let's see what you need.
```